samples/client/petstore/csharp-netcore/OpenAPIClientCore/src/Org.OpenAPITools/Model/Cat.cs

```diff
@@ -30,6 +31,9 @@ namespace Org.OpenAPITools.Model
     /// Cat
     /// </summary>
     [DataContract]
+    [JsonConverter(typeof(JsonSubtypes), "ClassName")]
+    [JsonSubtypes.KnownSubType(typeof(BigCat), "BigCat")]
+    [JsonSubtypes.KnownSubType(typeof(BigCat), "BigCat")]
```

Comment:
This is duplicated because the mustache file has:
```
    {{#children}}
    [JsonSubtypes.KnownSubType(typeof({{classname}}), "{{^vendorExtensions.x-discriminator-value}}{{name}}{{/vendorExtensions.x-discriminator-value}}{{#vendorExtensions.x-discriminator-value}}{{{vendorExtensions.x-discriminator-value}}}{{/vendorExtensions.x-discriminator-value}}")]
    {{/children}}
    {{#mappedModels}}
    [JsonSubtypes.KnownSubType(typeof({{{modelName}}}), "{{{mappingName}}}")]
    {{/mappedModels}}
```
My update makes it so all of our desired info is in mappedModels.
How should I handle these?
- {{#vendorExtensions.x-discriminator-value}}
- {{#children}} (child models that allOf inherit from the current schema)
My update moves all mapping information into mappedModels including children. I am not sure if it also includes #vendorExtensions.x-discriminator-value.